api/handlers_metrics.go

```diff
@@ -258,6 +258,95 @@ func (s *server) GetS3MetricsURLHandler(w http.ResponseWriter, r *http.Request)
 	w.Write(meta)
 }
 
+// GetRDSMetricsURLHandler gets RDS metrics from cloudwatch and returns a link to the image
+// Can use DBInstanceIdentifier or DBClusterIdentifier
+func (s *server) GetRDSMetricsURLHandler(w http.ResponseWriter, r *http.Request) {
+	w = LogWriter{w}
+	vars := mux.Vars(r)
+	account := vars["account"]
+	queryType := vars["type"]
+	instanceId := vars["id"]
+
+	cwService, ok := s.cloudwatchServices[account]
+	if !ok {
+		msg := fmt.Sprintf("cloudwatch service not found for account: %s", account)
+		handleError(w, apierror.New(apierror.ErrNotFound, msg, nil))
+		return
+	}
+	log.Debugf("found cloudwatch service %+v", cwService)
+
+	resultCache, ok := s.resultCache[account]
+	if !ok {
+		msg := fmt.Sprintf("result cache not found for account: %s", account)
+		handleError(w, apierror.New(apierror.ErrNotFound, msg, nil))
+		return
+	}
+	log.Debugf("found cost explorer result cache %+v", *resultCache)
+
+	queries := r.URL.Query()
+	metrics := queries["metric"]
+	if len(metrics) == 0 {
+		handleError(w, apierror.New(apierror.ErrBadRequest, "at least one metric is required", nil))
+		return
+	}
```

Comment:
just a note - I was just poking around at this and I think it could be moved into the `parseQuery` func too... not sure why I didn't.  I can do it more holistically later though.

Reply:
:+1: